webapp/channels/src/actions/views/create_comment.tsx

```diff
@@ -63,10 +72,27 @@ export function submitPost(channelId: string, rootId: string, draft: PostDraft):
             pending_post_id: `${userId}:${time}`,
             user_id: userId,
             create_at: time,
-            metadata: {},
```

Comment:
The metadata is used, if I am not mistaken, only for priority messages, which can only be root posts. Since this function (despite its name) is only used for creating comments, there was no need for metadata. After the changes, both create post and create comment use the same function, if I remember correctly my changes.